<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>속성 설명</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>331</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QLabel" name="change_label">
      <property name="text">
       <string>변형 후</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="originalImage_label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="original_label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string>원본</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="changeImage_label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>331</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="title_label">
      <property name="text">
       <string>horizontal_flip</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="example_label">
      <property name="text">
       <string>수평으로 뒤집는다.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>311</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>horizontal_flip</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>width_shift_range</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>height_shift_range</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>rotation_range</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>shear_range</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>zoom_range</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>vertical_flip</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>fill_mode : constant</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>fill_mode : nearest</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>fill_mode : reflect</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>fill_mode : wrap</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
